session.go

```diff
@@ -152,6 +152,9 @@ func (sess *session) handleRequests(reqs <-chan *gossh.Request) {
 			var kv = struct{ Key, Value string }{}
 			gossh.Unmarshal(req.Payload, &kv)
 			sess.env = append(sess.env, fmt.Sprintf("%s=%s", kv.Key, kv.Value))
+			if req.WantReply {
+				req.Reply(true, nil)
+			}
 		case "pty-req":
```

Comment:
Maybe the simplest solution is to ignore/deny (reply false) any more pty requests...